queries.sql

```diff
@@ -31,3 +31,20 @@ WHERE name NOT IN ('Gabumon');
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts > 0;
```

Comment:
**Changes Required**
- [ ] We need to select where animals have `never` escaped. Apparently this query returns animals that have escaped